src/h_vialib/client.py

```diff
@@ -56,7 +57,10 @@ def __init__(self, secret, service_url=None, html_service_url=None):
             "via.external_link_mode": "new-tab",
         }
 
-    def url_for(self, url, content_type=None, options=None, blocked_for=None):
+    # pylint:disable=too-many-arguments
```

Comment:
I reckon this might keep growing and have a similar interface to requests.request.

Not a problem IMO.

Reply:
:+1: